tests/test_views.py

```diff
@@ -65,6 +65,24 @@ def setUp(self):
         patcher.start()
         self.addCleanup(patcher.__exit__, None, None, None)
 
+    def test_access_to_staff_only(self):
```

Comment:
Can we rename this as it doesn't test the staff access, but the restricted access for non-staff users. Or we can create another staff user and test the permissions on it.

Reply:
Thanks! You're right this is kind of doing the opposite of what it says.